### PR TITLE
[indigo] Changelog for 0.7.4 (preparing to fix #386)

### LINKED ISSUE #386: [Indigo] 0.7.3 release build failure on ROS buildfarm
Related to #100

Looks like the build farm is failing to build `moveit` 0.7.3-0 particularly `moveit_kinematics` e.g. [this job](http://build.ros.org/view/Ibin_uT64/job/Isrc_uT__moveit_kinematics__ubuntu_trusty__source/5/console):

```
12:12:14 Traceback (most recent call last):
12:12:14   File "/tmp/ros_buildfarm/scripts/release/get_sources.py", line 48, in <module>
12:12:14     sys.exit(main())
12:12:14   File "/tmp/ros_buildfarm/scripts/release/get_sources.py", line 44, in main
12:12:14     args.os_name, args.os_code_name, args.source_dir)
12:12:14   File "/tmp/ros_buildfarm/ros_buildfarm/sourcedeb_job.py", line 59, in get_sources
12:12:14     (source_version, pkg_version))
12:12:14 RuntimeError: The cloned package version from the GBP (0.9.3-0trusty) does not match the expected package version from the distribution file (0.7.3-0)
12:12:14 Build step 'Execute shell' marked build as failure
```
Indeed [I had to change the version of the package](https://github.com/ros-planning/moveit/commit/746dbf9dcc663751e8b2eb3d68b09c080101dcd8#diff-2606211cb8b44b1187b384e29859dd8dL4) (i.e. downgrade from 0.9.3 to 0.7.2. This package was maintained with different version number and moved into this repo recently) in order for it to align with others in the same repo (if I remember correctly this was needed by `catkin_prepare_release` or `bloom`, or both?). 

I'm not sure what to do to fix this case. @tfoote @dirk-thomas I'd appreciate if you could help.

### PR DESCRIPTION
**Pull request by 130s**
tldr; I would propose we might as well move forward making a new release 0.7.4 even <s>without updating changelog</s> without adding changelog entries for the actual changes since the last valid release in June 2016. This should help fixing #386, which is a priority.

- Remove changelog entries with higher versions than we're releasing (pointed out at https://github.com/ros-planning/moveit/issues/386#issuecomment-268689110)
- Even doing so catkin_generate_changelog got stuck so that it didn't generate entries since the last release (that was made in 201606).
- Decided to bump version of all packages uniformely to 0.7.4 in the hope for catkin_generate_changelog to function, only to find that `catkin_generate_changelog` still doesn't pick up any commit logs since 201606.
- New release 0.7.4
  - It's sad to not see changelog entry for the new version, but #386 needs fixed immediately.
  - Too difficult to workaround catkin_generate_changelog issue above.
  - Upon successful release I @130s will manually write notable updates.

### CODE REVIEW

**Comment by 130s** [comment]
> tldr; I would propose we might as well move forward making a new release 0.7.4 
- x even without updating changelog.
- o even without adding changelog entries for the actual changes since the last valid release in June 2016.

**Comment by 130s** [comment]
Travis passed. 
I'll go ahead merge without getting approval (as always for administrative changes exception described in ["No Direct Commits" rule](http://moveit.ros.org/documentation/contributing/pullrequests/)).

**Comment by davetcoleman** [comment]
retroactive +1
